pennylane/math/quantum.py

```diff
@@ -164,6 +169,167 @@ def marginal_prob(prob, axis):
     return np.flatten(prob)
 
 
+def _density_matrix_from_matrix(density_matrix, wires, check_state=None):
+    """Compute the density matrix from a state vector.
+
+    Args:
+        density_matrix (tensor_like): 1D tensor state vector. This tensor should be of size ``(2**N,)`` for some
+            integer number of wires``N``.
+        wires (list(int)): List of wires (int) in the subsystem.

```

Comment:
As mentioned in the other PR, we should call these something like `subsystem_indices` since they have to correspond to how to index into the state.  When I read wires, I think "general label".

Reply:
I am changing it here as well 👍